lib/Twig/Node/ForLoop.php

```diff
@@ -13,6 +13,8 @@
  * Internal node used by the for node.
  *
  * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @internal
```

Comment:
should this really be marked as ``@internal`` ? Node visitors may need to be aware of it.